bebop_driver/src/bebop_driver_nodelet.cpp

```diff
@@ -232,6 +235,23 @@ void BebopDriverNodelet::CmdVelCallback(const geometry_msgs::TwistConstPtr& twis
   }
 }
 
+void BebopDriverNodelet::CmdMoveByCallback(const geometry_msgs::TwistConstPtr& twist_ptr)
```

Comment:
I'm not sure, if I understood your comment correctly. Pleas let me know, if it's the case.